BoxBrowseSDK/BoxBrowseSDK/Controllers/BOXItemsViewController.m

```diff
@@ -146,8 +146,7 @@ - (void)setupNavigationBar
     
     if (shouldShowCloseButton) {
 
-        NSString *closeButtonTitle = [[NSString alloc] init];
-
+        NSString *closeButtonTitle;
```

Comment:
It doesn't have nullability specified, but in practice it is non-null, if the delegate implements it.